libnopegl/src/utils.h

```diff
@@ -158,4 +162,20 @@ char *ngli_numbered_lines(const char *s);
 int ngli_config_copy(struct ngl_config *dst, const struct ngl_config *src);
 void ngli_config_reset(struct ngl_config *config);
 
+static inline uint32_t ngli_clz(uint32_t x)
```

Comment:
Can you add a comment about ub with x=0?

Reply:
Done for clz and log2.